### ISSUE TITLE
readme

### ISSUE THREAD

**Issue by Finnfuture** [hycarbon-b/fcb_v1#1]
python train.py --dataset Kvasir --data-root /path/to/dataset/ --epochs 200 --batch-size 4 --learning-rate 1e-4 --learning-rate-scheduler true --learning-rate-scheduler-minimum 1e-15 --multi-gpu false
Argument Details
--dataset: Specify the dataset to use (choices: Kvasir, CVC, chest, siim).
--data-root: Provide the root directory of the dataset.
--epochs: Number of training epochs (default: 200).
--batch-size: Batch size for training (default: 16).
--learning-rate: Initial learning rate (default: 1e-4).
--learning-rate-scheduler: Use learning rate scheduler (choices: true, false, default: true).
--learning-rate-scheduler-minimum: Minimum learning rate for the scheduler (default: 1e-15).
--multi-gpu: Use multiple GPUs for training (choices: true, false, default: false).

python predict.py --train-dataset siim --test-dataset siim --data-root /path/to/dataset/
Argument Details
--train-dataset: Specify the training dataset to use (choices: Kvasir, CVC, chest, siim).
--test-dataset: Specify the test dataset to use (choices: Kvasir, CVC, chest, siim).
--data-root: Provide the root directory of the dataset.
PS. this predict.py need the truth lables'file. If you have none ,you need rewrite this file

Please do not use the default path, you need add a forward slash at the end of the file path, for example: /FCBFormer/predict.py --train-dataset="chest" --test-dataset="chest" --data-root="/code/fcbformer/zhou/originalfile/chest/".don‘t use stand

**Comment by steve-zeyu-zhang** [comment]
Thanks Zhou, updated in readme.
